### DISCUSSION TITLE
Navigation group children all active when selecting parent

### DISCUSSION THREAD

**Discussion by micraux** [filamentphp/filament#12218] [Help]
### Package

Panel builder

### Package Version

v3.2.63

### How can we help you?

Hello,

In the navigation, when I click on a parent item, all his children items are active.

I would prefer only the parent to be active, the same way only the selected child is active when I click on it.

![group-items](https://github.com/filamentphp/filament/assets/46725831/28245116-c5d6-40b2-8bf8-4dba479716a4)

Is it the normal behavior or am I missing something?

**Comment by dissto** [comment]
Are you using a [custom navigation](https://filamentphp.com/docs/3.x/panels/navigation#registering-custom-navigation-items)?

**Reply by micraux** [reply]
Hello,

No, but you put me in the right direction, thanks!

Turns out I modified `.fi-sidebar-item-active a` in my theme.css.
